Updated script to accommodate multiple instances and regions

teusink opened the pull request:
Updated script to accommodate multiple instances and regions in one AWS-account. Tested it and it seems to work :). Also moved the instance names and region to the top.

teusink committed:
Updated script to accommodate multiple instances and regions

Updated script to accommodate multiple instances and regions in one AWS-account. Tested it and it seems to work :). Also moved the instance names and region to the top.

vidanov reviewed (changes requested):
Hey! 

Awesome! I appreciate it very much.
Thank you a lot for the great changes! Could you please just to add an example of the usage for the var instances in the comments - where to find them and how it may look.  //  http://take.ms/dChbs

I want to keep it simple and fast to implement. 

Best regards,
Alexey

teusink committed:
Added reference to find instance name and region

// Your instance name and region can be found here (see image): http://take.ms/dChbs

teusink commented:
Hi, you're welcome! I have added this line in patch-1:
`// Your instance name and region can be found here (see image): http://take.ms/dChbs`

Is this what you meant? And thanks for your script, it helps me allot!
